api/base/serializers.py

```diff
@@ -68,6 +68,12 @@ def to_representation(self, obj):
             ret['self'] = obj.get_absolute_url()
         return ret
 
+
+class LinksFieldNoSelfLink(LinksField):
```

Comment:
Let's make this one the LinksField, and the other LinksFieldWithSelfLink. It's a minor thing, but I think making the default case be the simplest one will make it easier down the line.